### ISSUE TITLE
Awesome work!

### ISSUE THREAD

**Issue by beikov** [orioledb/orioledb#11]
Just wanted to let you know that I think this engine is amazing and I hope the it will one day be part of PostgreSQL Core.

**Comment by akorotkov** [comment]
Thank you very much! Being a part of PostgreSQL Core is our goal!

**Comment by ansarizafar** [comment]
I totally agree with @beikov.  @akorotkov  Is there any progress in thread based connections? I think with other benefits  a built-in  HTTP server can also solve Postgres connection issue.

**Comment by postgresnx** [comment]
Please take into consideration that this is a storage engine and does not modify the parts of the Postgres core that handle connections etc. Thread-based connections is an ongoing discussion in the core community and actively being considered. But at this time, there is no public commitment for this. Your ideas and requests are appreciated. Thanks.

**Comment by ImreSamu** [comment]
OrioleDB in the "Technology Radar Volume 27" ( Published : Oct 26, 2022 )
* https://www.thoughtworks.com/radar/platforms/orioledb
* PDF: https://www.thoughtworks.com/content/dam/thoughtworks/documents/radar/2022/10/tr_technology_radar_vol_27_en.pdf

**Comment by postgresnx** [comment]
thank you for this, you made my day :) I will think of a special prize for your contribution. please email me :)
